xml_inscriptions/transcribed/KY.Lou.SAM.L.1929.17.485A-C.xml

```diff
@@ -104,18 +111,25 @@
       
         <body>
         <div type="edition">
-         <!--The marked up transcription goes here, surrounded by <ab> </ab> elements-->
-         </div>
+           <ab>
+              <lb n="1"/><persName><name type="praenomen" key="Sextus"><expan><abbr><unclear>S</unclear>ex</abbr><ex>tus</ex></expan></name>
+                 <name type="gentilicium" key="Lartidius">Lartidius</name> <g type="mulieris"/> <expan><abbr>l</abbr><ex>ibertus</ex></expan>
+              <lb n="2"/><name type="cognomen" key="Trypho">Trypho</name></persName>;
+              <lb n="3"/><persName><name type="gentilicium" key="Lartidia"><supplied reason="lost">L</supplied>artidia</name> 
+                 <persName><name type="praenomen" key="Sextus"><expan><abbr>Sex</abbr><ex>ti</ex></expan></name></persName> 
+                 <expan><abbr>l</abbr><ex>iberta</ex></expan> <name type="cognomen" key="Donata">Do<hi rend="ligature">na</hi>ta</name></persName> 
```

Comment:
I think you're missing a character in your ligature based on the commentary below?